src/scripts/webpack/webpack.base.js

```diff
@@ -10,24 +10,30 @@ const JSLoader = t.enums.of(['babel', 'typescript'], 'JSLoader');
 
 export default ({ config, paths, NODE_ENV, jsLoader = JSLoader('babel') }) => {
 
-  const preLoaders = NODE_ENV === 'development' ? [
+  const preLoaders = [
     // linting with eslint
     {
       test: /\.jsx?$/, // test for both js and jsx
-      loader: 'eslint',
+      use: [{
+        loader: 'eslint-loader',
+        options: {
+          failOnError: NODE_ENV === 'production',
+          failOnWarning: NODE_ENV === 'production'
+        }
+      }],
       include: paths.SRC,
       exclude: paths.ASSETS
     }
-  ] : [];
+  ];
 
```

Comment:
I would just move the "preloaders" down with the rest of the rules, now that they're not optionally included